

<launch>
  <node pkg="pointcloud_to_grid" type="pointcloud_to_grid_node" name="ptg" output="screen" >
    <!-- center position xy -->
    <param name="position_x" value="0"/>
    <param name="position_y" value="0"/>
    <!-- size of a grid cell -->
    <param name="length_x" value="100"/>
    <param name="length_y" value="100"/>
    <!--
    <param name="cloud_in_topic" value="/os_cloud_node/points"/>
    -->
    <param name="cloud_in_topic" value="/filtered_points"/>
    <!--
    <param name="frame_out" value="os1_sensor"/>
    -->
    <param name="maph_topic_name" value="occupancy_grid"/>
  </node>
  <!-- <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconf" output="screen" /> -->
</launch>
